pep-9999.rst

```diff
@@ -87,6 +87,57 @@ Supported Constructs
 This sections lists constructs that compliant type checkers are expected
 to accept. Type stub authors can safely use these constructs.
 
+Aliases
+-------
+
+Type checkers should accept module-level and class-level aliases, e.g.::
+
+  IntList = List[int]
+
+  class C:
+    def f(self) -> int: ...
+    g = f
+
+An alias to a type may contain type variables, in which case all type variables
+must be substituted when the alias is used::
+
+  _K = TypeVar("_K")
+  _V = TypeVar("_V")
+  _MyMap = Dict[str, Dict[_K, _V]]
+
+  # either concrete types or other type variables can be substituted
+  def f(x: _MyMap[str, _V]) -> _V: ...
+  # explicitly substitute in Any rather than using a bare alias
+  def f(x: _MyMap[Any, Any]) -> Any: ...
```

Comment:
Since this is part of the same example, I'd use a different name for the second function.